apps/dashboard/app/helpers/batch_connect/sessions_helper.rb

```diff
@@ -72,29 +72,24 @@ def created(session)
     render(partial: 'batch_connect/sessions/card/created', locals: { session: session })
   end
 
-  def session_time(session)
+  def session_time_helper(session)
```

Comment:
Instead of renaming this `helper` can you keep `session_time` and rename the other method `render_session_time` (you'll have to change the caller too).